linuxrc_hostname.md

```diff
@@ -1,31 +1,76 @@
 # hostname setting in installation/rescue system
 
-There are 3 cases:
+The hostname can be set (in order of precedence):
 
-1. no hostname is set
+1. via `hostname` boot option
+2. via DHCP (either from HOSTNAME or FQDN)
+3. Default hostname set by Yast
 
-2. hostname is set via dhcp (either from HOSTNAME or FQDN)
+linuxrc only writes `/etc/install.inf`.
+After that, Yast takes over.
+see https://github.com/yast/yast-network/blob/master/doc/hostname.md
 
-3. hostname is set via `hostname` boot option
+## installation system
 
-linuxrc defaults to `install`, which may be overridden by dhcp (2) or user
-(3). (3) takes precedence over (2).
+- `etc/hostname` is an empty file at the beginning of the installation.
+- `linuxrc` doesn't write anything to `/etc/hostname`.
+- `linuxrc` writes by default `/etc/install.inf::SetHostname=1`.
 
-## installation system
+## if `hostname` boot option is used
+
+1. linuxrc takes the value of `hostname` boot option and writes it to
```

Comment:
It also sets the current hostname with the one given

Reply:
Is current hostname a synonym of transient hostname?

Reply:
well ... yes and no. I'll try to simplify it ... Basically you have "static hostname" - the hostname which is set according to /etc/hostname and "transient hostname" - hostname which is e.g. set from dhcp or e.g. using ```hostname <hostname>```. 

What is important here is that linuxrc sets transient hostname which means that the hostname is not written into ```/etc/hostname/``` and will not survive reboot.

Sorry for being pedantic but we have had lot of issues with that during past years bcs of lot of misunderstanding.

Reply:
yes, but, what do you refer to as "current hostname"?